README.md

```diff
@@ -1,2 +1,40 @@
-# gaia-automation
-Charter for the Gaia Automation team
+Gaia Automation
+===============
+
```

Comment:
This is a good start. In addition to the big picture mission and summary, I'd like to see a section about the more concrete stuff we're doing over the second half of this year. Basically our Q3 goals and the likely goals for Q4 based on what we know today.